components/molecules/List.js

```diff
@@ -6,7 +6,7 @@ import PropTypes from "prop-types";
 export function List(props) {
   let opacity = 40;
   return (
-    <ol className={props.className}>
+    <ul className={props.className} role="presentation">
```

Comment:
My interpretation of the fix is that the `role="presentation"` should just be on the number, so rather than adding it in the `<ul>` it would be put in the `<span>` in each `<li>`

Reply:
Thank you Kyle, I have moved the role to ```<span>```